server/svix-server/src/queue/redis.rs

```diff
@@ -33,7 +33,7 @@ use axum::async_trait;
 
 use chrono::Utc;
```

Comment:
I'm still unclear about the relationship of all of these things.

For example, if I:
1. Add a message.
2. XREADGROUP it
3. XACX it

Will it show in XREADGROUP again? Will it show in XPENDING? Will it show in neither?

I get the idea of using XPENDING, it sounds clever and correct, but can you maybe document all of this in the PR description too (and thus in the commit message)?